src/server/chunk_server/run_chunk_server_main.cc

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
```

Comment:
can remove in another PR

Reply:
It's fine. Can do  this later. I had these comments since yesterday before you merged but didn't click on submit review. So all my comments were not submitted.